runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -19,11 +19,14 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
  *******************************************************************************/
+#include <cstring>
 #if !defined(JFRCONSTANTPOOLTYPES_HPP_)
 #define JFRCONSTANTPOOLTYPES_HPP_
 
 #include "j9cfg.h"
+#include "j9hypervisor.h"
```

Comment:
This leads to compile failures with UMA, which should be fixed by https://github.com/eclipse-openj9/openj9/pull/20164.